services/src/contexts/migrations/migration_0006_owner_role.rs

```diff
@@ -0,0 +1,26 @@
+use async_trait::async_trait;
+use tokio_postgres::Transaction;
+
+use crate::error::Result;
+
+use super::database_migration::{DatabaseVersion, Migration};
+
+/// This migration adds a check that there is only one owner per resource
+pub struct Migration0006OwnerRole;
+
+#[async_trait]
+impl Migration for Migration0006OwnerRole {
+    fn prev_version(&self) -> Option<DatabaseVersion> {
+        Some("0005_gbif_column_selection".into())
+    }
+
+    fn version(&self) -> DatabaseVersion {
+        "0006_owner_role".into()
```

Comment:
6 is now ebv.

You could either change it to 7 or say fuck it, no one ever used duplicate roles before now.

Reply:
514bf67c5358edad41c2fe7d90c047e98d7b6644